portfolio/src/main/webapp/script.js

```diff
@@ -48,3 +48,47 @@ function addPhoto() {
   imageContainer.innerHTML = '';
   imageContainer.appendChild(imgElement);
 }
+
+function getCommentSectionFromServer() {
+  fetch('/comment-section').then(response => response.json()).then((comments) => {
+    const commSection = document.getElementById('comment-list');
+    // Build the comment section.
+    comments.forEach((comm) => {
+      commSection.appendChild(createIndividualComment(comm));
+    });
+  });
+}
+
+// Format each particular comment to show up on page.
+function createIndividualComment(comment) {
+  const liElement = document.createElement('li');
+  // This container is used for showing the username.
+  const usernameElement = document.createElement('div');
+  usernameElement.setAttribute('class', 'username-container');
+  // This container is used for showing the text of the comment.
+  const commentElement = document.createElement('div');
+  commentElement.setAttribute('class', 'comment-text');
+  // Made an image represent the delete button.
+  const deleteButtonElement = document.createElement('img');
+  deleteButtonElement.setAttribute('class', 'delete-button');
+  deleteButtonElement.src = 'images/trash.png';
+  deleteButtonElement.addEventListener('click', () => {
+    console.log('here');
+    deleteComment(comment);
+    // Remove the comment from the DOM.
+    liElement.remove();
+  });
+
+  usernameElement.innerText = comment.username + ' wrote:';
+  commentElement.innerText = comment.text;
+  liElement.appendChild(deleteButtonElement);
+  liElement.appendChild(usernameElement);
+  liElement.appendChild(commentElement);
+  return liElement;
+}
+
+function deleteComment(comment) {
+  const params = new URLSearchParams();
+  params.append('id', comment.id);
+  fetch('/delete-comment', {method: 'POST', body: params});
```

Comment:
Also here, I tried to set the method to be DELETE, but I think the mistake was related to how I set the body part.